components/org.wso2.carbon.identity.conditional.auth.functions.http/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/http/AbstractHTTPFunction.java

```diff
@@ -161,10 +410,8 @@ private boolean isValidRequestDomain(URI url) {
             return true;
         }
 
-        if (LOG.isDebugEnabled()) {
-            LOG.debug("Domain: " + domain + " extracted from url: " + url.toString() + " is not available in the " +
-                    "allowed domain list: " + StringUtils.join(allowedDomains, ','));
-        }
+        LOG.error("Domain: " + domain + " extracted from url: " + url.toString() + " is not available in the " +
+                "allowed domain list: " + StringUtils.join(allowedDomains, ','));
```

Comment:
It's not a good idea to change this debug log to an error. Looks like that's been added with the fact to not to reveal information

Reply:
Resolved with https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/157/commits/778ee19b0b9677f6d2cf96e829414272e804de01